### ISSUE TITLE
Attach Interrupt to RISING or FALLING

### ISSUE THREAD

**Issue by xavierzip** [theoctal/octalbonescript#40]
Hi,
Last time I reported an issue with attaching a pin to b.CHANGE. After the fixing bug the CHANGE mode functions correctly. Recently I change some of the pins to interrupt on RISING and FALLING edge. The interrupt routine actually triggers on both RISING and FALLING. In a nut shell, all interrupts are effectively on CHANGE mode but not RISING or FALLING. 

Here are my codes:

```javascript

var bone 	= require('octalbonescript');
bone.pinMode('P9_14', bone.INPUT_PULLUP, function(err, pin){
	bone.attachInterrupt('P9_14', bone.RISING, function(atItrperror, resp){
		// bone.digitalRead('P9_14', function(dRerr, value){
			console.log(resp.value)
		// })
	})
})

```
Could you help me verify whether you can duplicate this issue?

Thank you.

**Comment by adityapatadia** [comment]
OBS uses epoll nom module to detect interrupts. I have made slight modifications to code according to epoll documentation. Please update OBS and use code I provided below. Let me know if the issue still persists. Unfortunately, I currently don't have any means to test interrupts. 

```javascript

var bone = require('octalbonescript');
bone.attachInterrupt('P9_14', bone.RISING, function(atItrperror, resp){
    console.log(resp.value);
});

```

**Comment by adityapatadia** [comment]
Can you confirm is this is fixed?

**Comment by xavierzip** [comment]
Hi,
I am sorry that, I was really for the past few days. Finally I got time to verify the library again. I realized that actually the library before your modification works fine with RISING and FALLING. After changing a push button, it only outputs 1 for RISING and 0 for FALLING. I think it could be the button issue which creates lots of noisy signals as a result the library could not handle it. 

Thank you so much for your time in looking into this.
